sentences/de/homeassistant_HassStartTimer.yaml

```diff
@@ -0,0 +1,17 @@
+language: de
+intents:
+  HassStartTimer:
+    data:
+      - sentences:
+          - "<timer_duration> Timer"
+          - "Timer für <timer_duration>"
+          - "<timer_duration> Timer für {timer_name:name}"
+          - "Timer für <timer_duration> namens {timer_name:name}"
+          - "<timer_set>[ einen] <timer_duration> Timer"
+          - "<timer_set>[ einen] Timer für <timer_duration>"
+          - "<timer_set>[ einen] <timer_duration> Timer namens {timer_name:name}"
+          - "<timer_set>[ einen] Timer namens {timer_name:name} für <timer_duration>"
+          - "<timer_set>[ einen] Timer für <timer_duration> namens {timer_name:name}"
+        requires_context:
```

Comment:
just noticed, that only the start timer sentences are area-aware - is that intentional?

Reply:
I think so as these timers (once created) are bound to the satellite they were created on. "think" because there are only few details on the new timers feature. rc-release notes for tomorrow don't even include a mention of timers: https://rc.home-assistant.io/blog/2024/05/29/release-20246/

Reply:
yeah ... the release notes are mostly focused on usability and new features or integrations.

have placed a question in the discussion https://github.com/home-assistant/intents/discussions/2193#discussioncomment-9669915 how it's intended to work

Reply:
now I get it ... we have to keep the area-context only for the start sentences :+1: